syntax = "proto3";

package test.integration.filters;

message CrashFilterConfig {
  bool crash_in_encode_headers = 1;
  bool crash_in_encode_data = 2;

  bool crash_in_decode_headers = 3;
  bool crash_in_decode_data = 4;
  bool crash_in_decode_trailers = 5;
  bool crash_in_decode_metadata = 6;
}
